Microsoft.DotNet.Interactive/Commands/SubmitCode.cs

```diff
@@ -16,6 +16,8 @@ public class SubmitCode : KernelCommandBase
             SubmissionType = submissionType;
         }
 
+        public bool AllowStdin { get; set; }
```

Comment:
Let's keep this on the context, not the command.

Reply:
It's handled in the context now.